runtime/src/main/java/io/novafoundation/nova/runtime/extrinsic/signer/NovaSigner.kt

```diff
@@ -23,4 +26,8 @@ interface FeeSigner : NovaSigner {
      * It might not be equal to [actualFeeSignerId] if [Signer] modifies the payload
      */
     suspend fun requestedFeeSignerId(chain: Chain): AccountId
+
+    override suspend fun modifyPayload(payloadExtrinsic: SignerPayloadExtrinsic): SignerPayloadExtrinsic {
+        throw NotImplementedError("This method should not be called")
+    }
```

Comment:
Looks like it is reasonable now to create a separate interface for real signers to expose this method there